doc/doxyfile.in

```diff
@@ -280,7 +280,7 @@ PERL_PATH              = /usr/bin/perl
 #---------------------------------------------------------------------------
 # Configuration options related to the dot tool
 #---------------------------------------------------------------------------
-CLASS_DIAGRAMS         = YES
+CLASS_DIAGRAMS         = NO
```

Comment:
Why?

Reply:
This was my temporary change to make doxygen generation faster, now reverted